Library/Homebrew/macos_version.rb

```diff
@@ -19,6 +19,7 @@ def initialize(version)
 
   # NOTE: When removing symbols here, ensure that they are added
   #       to `DEPRECATED_MACOS_VERSIONS` in `MacOSRequirement`.
+  # NOTE: Changes to this list must match `macos_version_name` in `cmd/update.sh`.
```

Comment:
Not blocking for this PR but might be nice to just pass these values through so they aren't duplicated or read from a shared location easily parseable by both.